test/mod/test_buildroot.py

```diff
@@ -135,5 +143,41 @@ def test_selinuxfs_ro(self):
                    "/sys/fs/selinux",
                    "ro"]
 
-            r = root.run(cmd, monitor, readonly_binds=ro_binds)
+            r = root.run(cmd, monitor, timeout, readonly_binds=ro_binds)
             self.assertEqual(r.returncode, 0)
+
+    def test_timeout(self):
+        runner = detect_host_runner()
+        libdir = os.path.abspath(os.curdir)
+        var = pathlib.Path(self.tmp.name, "var")
+        var.mkdir()
+
+        with BuildRoot("/", runner, libdir, var) as root:
+
+            READ_ONLY = select.POLLIN | select.POLLPRI | select.POLLHUP | select.POLLERR
+            poller = select.poll()
+
+            proc = subprocess.Popen(['python3', libdir + '/stages/org.osbuild.test.timeout'], stdout = subprocess.PIPE)
```

Comment:
Does this stage exist? Forgot to commit it? I assume it simply blocks forever?

Reply:
Forgot to commit it :upside_down_face: 

Added it now, hopefully it will make the tests make more sense